### PR TITLE
Run VeyMont in CI

### PR DESCRIPTION
**Pull request by bobismijnnaam**

### CODE REVIEW

File: examples/veymont-global-programs/TicTacToe.pvl [hunk 1]
```diff
@@ -1,6 +1,5 @@
 // -*- tab-width:4 ; indent-tabs-mode:nil -*-
-//:: cases TicTacToeInt TicTacToeMatrix
-//:: tools veymont
```
Comment (raulmonti) [inline examples/veymont-global-programs/TicTacToe.pvl:3]: why did you remove this line?

**Review by raulmonti — APPROVED** [review]
Looks nice to me (loved the sh code :D)

File: examples/veymont-global-programs/TicTacToe.pvl (hunk 1 above)
Comment (bobismijnnaam) [reply examples/veymont-global-programs/TicTacToe.pvl:3]: The global programs are run with the script in the util folder, so they do not have to be included in vercor's own test suite mechanism (the one triggered by passing the flag ```--test=./examples```). Removing the "tools" directive ensures it's not included in the test suite.

Comment (bobismijnnaam) [reply examples/veymont-global-programs/TicTacToe.pvl:3]: The script is at the bottom of the "files changes" page of this PR, it's called `veymont-2stage-tests.sh`.

Comment (raulmonti) [reply examples/veymont-global-programs/TicTacToe.pvl:3]: yes sorry, I realised that it might be that but did not remove the comment, the thing is that as you mention the script was at the bottom :D

**Comment by bobismijnnaam** [comment]
@raulmonti :) Let's hope the sh code will be short-lived and we can integrate veymont properly soon...

**Review by petravandenbos-utwente — APPROVED** [review]
Code looks good.
